pkgs/applications/networking/cluster/argo-rollouts/default.nix

```diff
@@ -0,0 +1,41 @@
+{ buildGoModule, lib, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "argo-rollouts";
+  version = "1.1.1";
+
+  src = fetchFromGitHub {
+    owner = "argoproj";
+    repo = "argo-rollouts";
+    rev = "v${version}";
+    sha256 = "0qb1wbv3razwhqsv972ywfazaq73y83iw6f6qdjcbwwfwsybig21";
+  };
+
+  vendorSha256 = "1rmw80lvs9qj2d195rvhzz8xrbx71dirv1c0grc113d18q609qwj";
+
+  # Disable tests since some test fail because of missing test data
+  doCheck = false;
+
+  runVend = true;
+
+  installPhase = ''
```

Comment:
https://github.com/NixOS/nixpkgs/blob/62aa9182375579b3e72b13e6cd98fcb5d36c6315/pkgs/applications/networking/cluster/argo/default.nix#L35

Should be able to use `subPackages` instead of `installPhase`.

Reply:
That's nice! I didn't know that.